catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -284,6 +285,27 @@ public UpdateResponse versionUpdate(UpdateResponse updateResponse)
     }
 
     /**
+     * Versions deleted {@link Metacard}s.
+     * @param deleteResponse Versions this responses deleted metacards
+     */
+    public void versionDelete(DeleteResponse deleteResponse) {
+        if (doSkip(deleteResponse)) {
+            return;
+        }
+        setSkipFlag(deleteResponse);
+
+        List<MetacardVersion> versionedMetacards = deleteResponse.getDeletedMetacards()
+                .stream()
+                .map(mc -> new MetacardVersion(mc,
+                        MetacardVersion.Action.DELETED,
+                        getSubject.get()))
+                .collect(Collectors.toList());
+
+        CreateResponse createResponse = getSystemSubject().execute(() -> catalogProvider().create(
+                new CreateRequestImpl(new ArrayList<>(versionedMetacards))));
```

Comment:
It does have to, because the Interface says `List<Serializable>` instead of `List<? extends Serializable>`. So we can't pass it a list of subclasses of serializable, we have to create a list of 'Serializable' objects which we can do by wrapping it in a new ArrayList.